### DISCUSSION TITLE
WinApps Taskbar Launcher [Legacy]

### DISCUSSION THREAD

**Discussion by KernelGhost** [winapps-org/winapps#124] [Ideas]
Hi everyone,

I've developed a simple application that lets you manage the 'RDPWindows' VM and launch applications directly from the system taskbar. Contributions, comments and suggestions are welcome.

You can find the project at https://github.com/KernelGhost/WinApps-Launcher.

Thanks!

**Comment by freechelmi** [comment]
Thanks ! We really need that , it won't Launch on my Ubuntu 22.04/ Gnome but I'll dig into it to see why

**Reply by KernelGhost** [reply]
@freechelmi Hope you are able to get it to work!
FYI, the project has moved to https://github.com/winapps-org/WinApps-Launcher.

**Reply by freechelmi** [reply]
> @freechelmi Hope you are able to get it to work! FYI, the project has moved to https://github.com/winapps-org/WinApps-Launcher.

Thanks , yes I was testing from there , I will open an issue , we really need that :-) 

OK might just be because I run on Wayland , we should at least give a warning at launch for this

**Reply by KernelGhost** [reply]
I have opened a PR you may be interested in [here](https://github.com/winapps-org/WinApps-Launcher/pull/13).
